go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/argoproj/argo-workflows/v3
 
 go 1.15
 
+replace github.com/argoproj/pkg v0.4.0 => github.com/simster7/pkg v0.2.1-0.20210205175442-61f69ff4a52f
```

Comment:
remove this

Reply:
Will do so once that PR is merged

Reply:
still pending?

Reply:
It had just been approved. Will remove this now